### ISSUE TITLE
Pip installation not working correctly

### ISSUE THREAD

**Issue by PSXBRosa** [ekiefl/pooltool#45]
I tried to install the project on two different machines using the pip install method and in both instances the package seemed to not be correctly installed. When trying to run the snippet provided on the readme section, the script failed to import any of the necessary classes, only having access to the constants.

Code example

```
#! /usr/bin/env python

import pooltool as pt

interface = pt.ShotViewer()

table = pt.PocketTable()
balls = pt.get_nine_ball_rack(table, ordered=True)
cue = pt.Cue(cueing_ball=balls['cue'])

# Aim at the head ball then strike the cue ball
cue.aim_at_ball(balls['1'])
cue.strike(V0=8)

# Evolve the shot
shot = pt.System(cue=cue, table=table, balls=balls)
shot.simulate(continuize=True)

interface.show(shot)
```

Error

```
Traceback (most recent call last):
  File "...\test.py", line 5, in <module>
    interface = pt.ShotViewer()
AttributeError: module 'pooltool' has no attribute 'ShotViewer'
```

**Comment by ekiefl** [comment]
Hi, thanks for checking out the project.

I'm sorry the PIP installation isn't working for you. I've noticed this myself on Windows (what OS are your machines?)

I plan to fix this for v0.2. Until then, I've updated the README accordingly in https://github.com/ekiefl/pooltool/commit/7b45c2cbfaeca04c742775cb3abb1024ff2f77f7

**Comment by PSXBRosa** [comment]
Hi

They are both windows machines.

Thank you for taking notice.

**Comment by HorseEars** [comment]
I can confirm that this is also happening on Ubuntu 20.04.5 LTS and other Linux machines.

**Comment by ekiefl** [comment]
Thanks for the report. In the meantime, follow the developer installation instructions. I really gotta push out 0.2.

**Comment by ekiefl** [comment]
@HorseEars, @PSXBRosa, can you guys please try and install on your linux/windows machines? I've released 0.2.0.

`pip install pooltool-billiards==0.2.0`

**Comment by ekiefl** [comment]
I'm closing this (for now), as I have heard through the grapevine that `pip` installation is now working on Linux and Windows.
